scripts/bump.js

```diff
@@ -0,0 +1,284 @@
+  packageJson.version = versionToBump;
+  writeJson(packageJsonPath, packageJson);
+};
+
+const bump = (packageName, versionToBump) => {
+  if (packageName === 'cli, core, schema') {
+    bumpMainPackages(versionToBump);
+    bumpMainPackagesForExampleApps(versionToBump);
+  } else {
+    bumpExtensionPackage(packageName, versionToBump);
+  }
+};
+
+const gitAdd = () => {
+  const result = spawnSync(
+    'git',
+    ['add', 'packages/*/package.json', 'example-apps/*/package.json'],
+    {
+      stdio: [0, 1, 2]
+    }
+  );
+
+  if (result.status !== 0) {
+    throw new Error('git-add failed');
+  }
+};
+
+const gitCommit = versionsToBump => {
+  const messageParts = Object.keys(versionsToBump).map(packageName => {
+    const fromVersion = PACKAGE_ORIG_VERSIONS[packageName];
+    const toVersion = versionsToBump[packageName];
+    return `${packageName} ${fromVersion} -> ${toVersion}`;
+  });
+  const message = 'Bump ' + messageParts.join(', ');
+  const result = spawnSync('git', ['commit', '-m', message], {
+    stdio: [0, 1, 2]
+  });
+
+  if (result.status !== 0) {
+    throw new Error('git-commit failed');
+  }
+};
+
+const gitTag = versionsToBump => {
+  const toVersions = Object.keys(versionsToBump).reduce(
+    (result, packageName) => {
+      const toVersion = versionsToBump[packageName];
+      if (packageName === 'cli, core, schema') {
+        result.cli = result.core = result.schema = toVersion;
+      } else {
+        result[packageName] = toVersion;
+      }
+      return result;
+    },
+    {}
+  );
+
+  Object.keys(toVersions).map(packageName => {
+    const version = toVersions[packageName];
+    const tag = `zapier-platform-${packageName}@${version}`;
+
+    const result = spawnSync('git', ['tag', '-a', tag, '-m', tag], {
+      stdio: [0, 1, 2]
+    });
+
+    if (result.status !== 0) {
+      throw new Error('git-tag failed');
+    }
+  });
+};
+
+const main = async () => {
+  try {
+    ensureMainPackageVersionsAreSame();
+  } catch (err) {
+    console.error(err.message);
+    return 1;
+  }
+
+  // Currently we bump cli, core, schema together
+  PACKAGE_ORIG_VERSIONS['cli, core, schema'] = PACKAGE_ORIG_VERSIONS.cli;
+
+  const packagesToBump = await promptPackagesToBump();
+  if (packagesToBump.length === 0) {
+    console.log('No packages selected, nothing to do here');
+    return 0;
+  }
+
+  const versionsToBump = {};
+  for (const packageName of packagesToBump) {
+    versionsToBump[packageName] = await promptVersionToBump(packageName);
+  }
+
+  Object.keys(versionsToBump).map(packageName => {
+    bump(packageName, versionsToBump[packageName]);
+  });
+
+  try {
+    gitAdd();
+    gitCommit(versionsToBump);
+    gitTag(versionsToBump);
+  } catch (err) {
+    console.error(err.message);
+    return 1;
+  }
+
+  console.log(
+    `\nDone! Now you can ${bold.underline('git push origin master --tags')}.`
+  );
+
+  return 0;
+};
+
+main().then(exitCode => {
+  if (exitCode) {
+    process.exit(exitCode);
```

Comment:
if there's an error partway through, it might be nice to include instructions to roll back changes (in this case, probably discarding unstaged git changes) so the command can be safely run again

Reply:
It'd be great to roll back automatically, but I'm taking a pass on that for now. Added a vague instruction in 1ca74d9.